python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -866,6 +871,76 @@ def _update_dist_lookup_table_vars(self, param_list, grad_list,
                 ]
         return param_list, grad_list
 
+    def _append_distributed_metric(self):
```

Comment:
can add a new transpiler? input a distributed program, and return a program with distributed metric?

Reply:
I think you are right, the distributed metrics are independent and multitudinous,   I will separate it from the current distributed program.